src/Tribe/Commerce/PayPal/Gateway.php

```diff
@@ -113,7 +116,12 @@ public function add_to_cart() {
 		$cart_url      = $this->get_cart_url( '_cart' );
 		$post_url      = get_permalink( $post );
 		$currency_code = trim( tribe_get_option( 'ticket-commerce-currency-code' ) );
-		$product_ids   = (array) $_POST['product_id'];
+
+		if ( isset( $_POST['tribe_ticket_id'] ) ) {
+			$product_ids = (array) $_POST['tribe_ticket_id'];
+		} elseif ( isset( $_POST['product_id'] ) ) {
+			$product_ids = (array) $_POST['product_id'];
```

Comment:
* :warning: Detected usage of a non-sanitized input variable: $_POST['product_id']
* :no_entry_sign: Processing form data without nonce verification.

<sub>hash: 54d835db2b57657137707949c08aa37c</sub>